### PR TITLE
Car details screen

### PR DESCRIPTION
**Pull request by TobiasBak**
Navigation between the different screen in the application has been implemented.

Themes are used as a context, which means it can be fetched from all screens using `const <variableName> = useContext(ThemeContext)`

### CODE REVIEW

**Comment by thor2304** [comment]
The CarDetailsScreen does not have the navigation in the bottom

File: screens/CarDetailScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import {getDefaultStyleSheet} from "../components/Stylesheet"
+import {View, Text, StyleSheet} from "react-native";
+import {ThemeContext} from "../components/ThemeContext";
+import {useContext} from "react";
+
+
+export function CarDetailScreen({route, navigation}: any) {
```
Comment (thor2304) [inline screens/CarDetailScreen.tsx:7]: Please add types for this.
I would also appreciate a short text that describes what the 2 parameters does and/or the format expected (should be clear from the type, but i hope it makes sense

**Comment by thor2304** [comment]
Never use `any` as the type for anything.
If `any` is used as the type: include a comment explaining why, and convert it to a typed variable as soon as possible (by checking for variables in the application logic. (I hope that makes sense)

File: screens/CarDetailScreen.tsx (hunk 1 above)
Comment (TobiasBak) [reply screens/CarDetailScreen.tsx:7]: I have removed the any type and written javadocs for the functions taking navigation or route as paremeters.

**Pull request merged**